samples/metrics_monitor/include/cttmetrics.h

```diff
@@ -70,7 +70,7 @@ typedef enum
     CTT_ERR_UNSUPPORTED                 = -3,  /* feature not supported */
     CTT_ERR_NOT_INITIALIZED             = -4,  /* the library not  initialized */
     CTT_ERR_NOT_FOUND                   = -5,  /* the specified object is not found */
-    CTT_ERR_ALREADY_INITIALIZED	        = -6,  /* the library already initialized */
+    CTT_ERR_ALREADY_INITIALIZED         = -6,  /* the library already initialized */
```

Comment:
This change should be the first, not the last.

Reply:
Again, this change does NOT help if it is not the first one! Please, do one of these things:
1. Either remove this change completely and fix a prev. patch where alignment is wrong.
2. Or move this change to be the FIRST in the series (I hope this will fix wrong alignments in the prev. patch)